libs/app-shell/src/lib/api/events/topics.service.ts

```diff
@@ -5,32 +5,31 @@ import { Injectable } from '@angular/core';
 /**
  * Its important to define topic first, which can set some parameters up front
  */
-@Injectable()
+@Injectable({ providedIn: 'root' })
 export class MessagingTopics {
```

Comment:
We dont want to expose this service yet outside adn we are not even exporting it to application. Should this still be providedin ROOT ? 


You are not able to create new topic or subsribe to topic if its not defined. and this is responsibility of internal api. I dont want application to create random topics.

Reply:
Yes, root. We don't have expose it, but we don't want to have duplicates